### ISSUE TITLE
DMN navigation controls lack keyboard focus indicator

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#4053]
### Describe the bug

As I tab through the DMN editor (decision table and literal expression views) with my keyboard I don't know what element has keyboard selection, and hence is activated if I _ENTER_:

![capture aLwMHU_optimized](https://github.com/camunda/camunda-modeler/assets/58601/12a4fd0d-1354-42a8-aa0c-ebe967ad1348)

### Steps to reproduce

1. Open Decision Table
2. Focus first cell
3. SHIFT+TAB to focus previous elements until you reach a navigation button
4. __See no focus indicator__

### Expected behavior

Keyboard focus is properly indicated.

### Environment

- OS: Linux
- Camunda Modeler Version: v5.19.0
- Execution Platform: Any
- Installed plug-ins: None

**Comment by smbea** [comment]
I was looking into this and we [leave the "powedered by" container but don't display the logo](https://github.com/camunda/camunda-modeler/blob/develop/resources/plugins/test/style.css). This makes it possible to focus on a mysterious element that opens this popup. Why don't we remove the whole thing?

https://github.com/camunda/camunda-modeler/assets/25825387/44c57527-3a76-486c-b4c2-91a30b8ae0d0

Ideally, it would work like:

https://github.com/camunda/camunda-modeler/assets/25825387/eafbe6c4-92b8-43a5-89d4-26a4f183e160

**Comment by nikku** [comment]
We should do whatever it takes to not allow this glitch to happen.

The plug-in you've shown is just there for tests; a focusable container _should not_ be shipped to released versions of the app.

**Comment by barmac** [comment]
This works fine on the demo page: https://demo.bpmn.io/dmn So we should probably remove the code which makes focus outline disappear :)

**Comment by smbea** [comment]
My bad, I was testing in development. In production that container is not there indeed. Will follow up to fix the focus styles
